kratos/includes/model_part_io.h

```diff
@@ -554,9 +548,9 @@ class KRATOS_API(KRATOS_CORE) ModelPartIO : public IO
 
     void ResetInput();
 
-    inline void CreatePartition(unsigned int number_of_threads,const int number_of_rows, DenseVector<unsigned int>& partitions);
-
+    inline void CreatePartition(unsigned int NumberOfThreads,const int number_of_rows, DenseVector<unsigned int>& partitions);
```

Comment:
As you changed the NumberOfThreads I would have changed the number_of_rows also

Reply:
```suggestion
    inline void CreatePartition(unsigned int NumberOfThreads,const int NumberOfRows, DenseVector<unsigned int>& partitions);
```